sdk/core/Azure.Core/src/Shared/ClientDiagnostics.cs

```diff
@@ -49,7 +43,7 @@ public ClientDiagnostics(ClientOptions options, bool? suppressNestedClientActivi
         ///  The default value could change in the future, the flag should be only set to false if suppression for the client
         ///  should never be enabled.</param>
         public ClientDiagnostics(string optionsNamespace, string? providerNamespace, DiagnosticsOptions diagnosticsOptions, bool? suppressNestedClientActivities = null)
-            : base(optionsNamespace, providerNamespace, diagnosticsOptions.IsDistributedTracingEnabled, suppressNestedClientActivities.GetValueOrDefault(false))
+            : base(optionsNamespace, providerNamespace, diagnosticsOptions.IsDistributedTracingEnabled, suppressNestedClientActivities.GetValueOrDefault(false), true)
```

Comment:
@lmolkova, this continues with the conversation about changing defaults for intentional breaking changes now that we are GA'ing the stable OTel functionality?

Reply:
there is no breaking change here - this is an internal shared class and adding parameters to it is not breaking. 
Or did I miss your point?

Reply:
Ah, no, I was primarily wondering if passing a new value was changing the behavior.  No worries either way, just looking to confirm my understanding.

Reply:
yes, this is changing the behavior  - if `true` is passed, then feature flag is no longer needed and tracing will be enabled without it.